public/css/verificationpages.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+    font-family: "montserrat-light";
+    src: url(../fonts/Montserrat-Light.ttf);
+  }
```

Comment:
You can remove this since we can access the font already through a global CSS file that was being included in head.php